tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/ParameterSymbolTest.java

```diff
@@ -103,6 +109,8 @@ public void testSymbolAtCursor(int line, int col, String paramName, TypeDescKind
                 {48, 20, "misc", ARRAY, REST, "anydata... misc"},
                 {53, 30, "p", TYPE_REFERENCE, REQUIRED, "Person p"},
                 {53, 44, "misc", ARRAY, REST, "anydata... misc"},
+                {62, 26, null, INT, REQUIRED, "int"},
+                //{65, 13, null, STRING, REQUIRED, "string"},
```

Comment:
Why is this case commented? If this is for a reason, shall we add a small comment and possibly point to the issue for the limitation?

Reply:
Yeah, I'll point this to a issue

Reply:
Not needed due to PR changes